src/crawlers/cheerio_crawler.js

```diff
@@ -115,13 +121,20 @@ const DEFAULT_OPTIONS = {
  *   The function receives the following object as an argument:
  * ```
  * {
- *   $: Cheerio, // the Cheerio object with parsed HTML
- *   html: String // the raw HTML of the page, lazy loaded only when used
+ *   $: Cheerio, // the Cheerio object with parsed HTML or XML
+ *   body: String|Object|Buffer // the request body of the web page
```

Comment:
As I mentioned in my Cheerio Crawler PR review, I'm not sure that it makes sense to have both raw and parsed values under body. IMO it should always be the raw version. String or Buffer. We could add a json getter which would JSON.parse the body if content type is JSON. I just can't wrap my head around having body as a raw HTML string once, and as parsed JSON the other time. @mtrunkat @jancurn @drobnikj

Reply:
Actually, you're right, I also dislike variables that can have too many possible types...

Reply:
BTW are we gonna keepthe `html` property there for backward compatibility?

Reply:
I'd say yes for a while in SDK with the deprecation warning. And yes, probably for a long time, in Scraper where we can't/don't want to release a major version just because of this.